### DISCUSSION TITLE
Logs empty using DigitalOcean one-click

### DISCUSSION THREAD

**Discussion by ebrawer** [mastodon/mastodon#22093] [Q&A] [answered]
4.0.2 via DO one-click. I am having trouble with streaming and searching for users on other servers. I can't check the logs because... they're empty. Anybody facing this or have an idea? Nothing turning them off in .env.production.

```
# journalctl -u mastodon-streaming
No journal files were found.
-- No entries --
# journalctl -u mastodon-web
No journal files were found.
-- No entries --
# journalctl -u mastodon-sidekiq
No journal files were found.
-- No entries --
```

**Comment by wiegelmann** [comment, marked as answer]
At DO journalctl is broken, fix it with "systemctl restart systemd-journald" - lessons learned after migrating a bunch of instances :)
